### ISSUE TITLE
[console] Status of this project

### ISSUE THREAD

**Issue by felmab** [hechoendrupal/DrupalConsole#4323]
A [blog post](https://webfutura.eu/drupal-custom-entities-drush) about a Drush entity generator [states that](https://webfutura.eu/drupal-custom-entities-drush#how-custom-entity-types-was-done-in-drupal-7---8) «Drupal console appeared as an alternative to drush to do all kinds of boilerplate code generation but if it is not abandoned now, it’s near to it».

There appears to be some activity, although the last release was made 11 months ago. Personally, I have stumbled upon some bugs with the entity generator in Drupal 9 (mainly missing or deprecated components like #4284) which can take some time to understand before you find the related issue, and can be rather tricky depending on the level  of expertise of the user. At this point, however, I believe the Drupal console entity generator is superior to the Drush one.

So, what is the status of this project? Is it still maintained? Will there be new releases?

**Comment by LedzZm** [comment]
I think since drush has adopted the sympfony command logic, there is not much merit in maintaining a project like drupal console. I am personally switching to drush, especially since the project seems abandoned. It would be nice though If we could get an update from the maintainers on the status.

Is drupal console discontinued/archived/on hold?

@jmolivas @enzolutions

**Comment by verbruggenalex** [comment]
I use drupal/console on all projects I work on. I don't see drush/drush having boilerplate code generation. I'm willing to dedicate my training budget towards helping out on the console.

And I see others jumping in as well: https://github.com/hechoendrupal/drupal-console/pull/4324 I think the status is that this project still has a lot of value.

**Comment by lhridley** [comment]
@verbruggenalex The command is `drush generate`.   It was introduced in Drush version 9.0.0-beta2, and incorporates the "[Drupal Code Generator](https://github.com/Chi-teck/drupal-code-generator)" project.

**Comment by JPustkuchen** [comment]
It would be wonderful if the Drupal Console maintainers would join Drush and put all the power into one tool. Better for DX and easier to learn... less dependencies etc...

Indeed, it would be nice to describe the state and future plan on the project page.

For example I just had this issue https://github.com/hechoendrupal/drupal-console/issues/2247 with old boilerplate code in drupal console... today I found out that `drush gen plugin-action` exists and solves it.

**Comment by DieterHolvoet** [comment]
I agree, an official stance would be nice. Merging Drush and Drupal Console would be a big win for the Drupal community IMO. I'd be willing to help transfer some commands that Drupal Console has but Drush doesn't.

**Comment by LOBsTerr** [comment]
It would be nice to see the list of missing Drush commands. So, we can a have plan what to do

**Comment by LOBsTerr** [comment]
@DieterHolvoet I am the only one, who is left on this project and obviously I don't want to dedicate to much time to a dead project. I am just trying to close critical issues. That is why I believe it would be great to focus on Drush. I maybe will do another release, but the problem is that I don't have the access to Drupal console core, which also has a lot of issues. I requested it. Let's see if I get an answer!

Great start! I think it could be

**Comment by DieterHolvoet** [comment]
Just did another update of [the spreadsheet](https://docs.google.com/spreadsheets/d/1DfdPwjiUBEYLJ7jR7l1SqoOzDTDxPQ5vS5FQO2MZQrY/edit#gid=0). Seems like by now, most useful commands exist in Drush or contrib. Might be a good point to start thinking about officially shutting down the project.
